.gitattributes

```diff
@@ -4,6 +4,7 @@ src/__Private/CodegenCLI.hack export-ignore
 src/codegen/data/ export-ignore
 tests/ export-ignore
 codegen/**/* linguist-generated
+codegen-no-rebuild/**/* linguist-generated
```

Comment:
We cannot `export-ignored` `codegen-no-rebuild/**`, because `is_compatible_schema_version` depends on `codegen-no-rebuild/latest_breaking_version.hack`